package/gluon-mesh-batman-adv-brmldproxy/Makefile

```diff
@@ -0,0 +1,36 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-mesh-batman-adv-brmldproxy
+
+include $(TOPDIR)/../package/gluon.mk
+
+define Package/$(PKG_NAME)
```

Comment:
OpenWrt's convention is never to use `$(PKG_NAME)` like this, and Gluon follows this convention.